base/bucket_gocb.go

```diff
@@ -914,20 +914,35 @@ func (bucket CouchbaseBucketGoCB) GetWithXattr(k string, xattrKey string, rv int
 
 		switch err {
 		case nil:
-			// Successfully retrieved doc and xattr.  Copy the contents into rv, xv and return.
+			// Successfully retrieved doc and (optionally) xattr.  Copy the contents into rv, xv and return.
 			err = res.Content("", rv)
```

Comment:
Good idea - I've renamed it for clarity and to ensure there's no scope confusion in future.